tutorial/practicals/LFRic/building_code/4_psydata/solutions/value_range_check_all_transform.py

```diff
@@ -35,18 +35,16 @@
 # Modified: R. W. Ford, STFC Daresbury Lab
 
 '''Python script intended to be passed to PSyclone's generate()
```

Comment:
"...passed to the psyclone command via the..."

Reply:
Done.